Makefile

```diff
@@ -603,8 +325,6 @@ minio-secrets:
 	@echo '[default]' > $(DIR)/tmp/secrets/aws-credentials
 	@echo "aws_access_key_id=`cat tmp/secrets/minio-access-key`" >> $(DIR)/tmp/secrets/aws-credentials
 	@echo "aws_secret_access_key=`cat tmp/secrets/minio-secret-key`" >> $(DIR)/tmp/secrets/aws-credentials
-	cp $(DIR)/tmp/ssl/selfsigned-site.crt $(DIR)/tmp/secrets/selfsigned-site-crt
```

Comment:
Have you checked that local minio works without ssl certs?

Reply:
Yes, by changing console-address to http://${CANDIG_DOMAIN:{MINIO_UI_PORT} the console can be reached without ssl

Reply:
It doesn't seem to need the console-address at all: it works without ssl even with the port set as 9090 as before, even without the MINO_CONSOLE_ADDRESS env var in the docker-compose. I think it was glitching before because of the compose-% target change.